### DISCUSSION TITLE
my 40-pin display compatibility regarding

### DISCUSSION THREAD

**Discussion by beyond-space21** [vroland/epdiy#324] [Q&A] [answered]
Hi, Iam using a *10.3"* e.paper display. which has *40 pins* in it. I have attached the pinout table. 

kindly please verify weather the display is compactible to the board.

![image](https://github.com/vroland/epdiy/assets/75003063/9d7bb6dc-6b84-4028-9c50-04e79b7db7ea)

*also it would be helpful if made a document regarding the E.paper communication protocol as most of the people end up in this repository for driving e.paper display from stcratch.*

Thankyou

**Comment by martinberlin** [comment, marked as answer]
Please check it yourself against [published schematic ](https://vroland.github.io/epdiy-hardware/build/boards/epdiy-v7_schematic.pdf)
<img width="363" alt="image" src="https://github.com/vroland/epdiy/assets/2692928/794ae11a-4b07-4cec-880e-d0b84d0a50d1">
It seems to match the pinout.
